src/picker.rs

```diff
@@ -252,6 +261,46 @@ fn guess_protocol() -> (ProtocolType, bool) {
     (ProtocolType::Halfblocks, is_tmux)
 }
 
+// Since only the latest preview version of Windows Terminal supports Sixels,
+// it is necessary to get the version and determine whether it supports Sixels or not.
+#[cfg(windows)]
+fn support_sixels() -> Result<bool> {
```

Comment:
You should be able to determine sixel support from the device attributes query, as is already done here:

https://github.com/benjajaja/ratatui-image/blob/e630e680a186fc0c44c43be0e6966889094e525e/src/picker.rs#L344-L346

That way you'll be able to support sixels in other terminals as well.